src/test-utils.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const FUNCTIONS = [
+  'LOWER',
+  'COUNT',
+  'SUM',
+  'MIN',
+  'MAX',
+  'AVG',
+  'APPROX_COUNT_DISTINCT',
+  'APPROX_COUNT_DISTINCT_DS_HLL',
+  'APPROX_COUNT_DISTINCT_DS_THETA',
+  'APPROX_QUANTILE',
+  'APPROX_QUANTILE_DS',
+  'APPROX_QUANTILE_FIXED_BUCKETS',
+  'BLOOM_FILTER',
+  'ABS',
+  'CEIL',
+  'EXP',
+  'FLOOR',
+  'LN',
+  'LOG10',
+  'POWER',
+  'SQRT',
+  'TRUNCATE',
+  'TRUNC',
+  'ROUND',
+  'MOD',
+  'SIN',
+  'COS',
+  'TAN',
+  'COT',
+  'ASIN',
+  'ACOS',
+  'ATAN',
+  'ATAN2',
+  'DEGREES',
+  'RADIANS',
+  'CONCAT',
+  'TEXTCAT',
+  'STRING_FORMAT',
+  'LENGTH',
+  'CHAR_LENGTH',
+  'CHARARACTER_LENGTH',
+  'STRLEN',
+  'LOOKUP',
+  'LOWER',
+  'PARSE_LONG',
+  'POSITION',
+  'REGEXP_EXTRACT',
+  'REPLACE',
+  'STRPOS',
+  'SUBSTRING',
+  'RIGHT',
+  'LEFT',
+  'SUBSTR',
+  'TRIM',
+  'BTRIM',
+  'LTRIM',
+  'RTRIM',
+  'UPPER',
+  'REVERSE',
+  'REPEAT',
+  'LPAD',
+  'RPAD',
+  'CURRENT_TIMESTAMP',
+  'CURRENT_DATE',
+  'DATE_TRUNC',
+  'TIME_FLOOR',
+  'TIME_SHIFT',
+  'TIME_EXTRACT',
+  'TIME_PARSE',
+  'TIME_FORMAT',
+  'MILLIS_TO_TIMESTAMP',
+  'TIMESTAMP_TO_MILIS',
+  'EXTRACT',
+  'FLOOR',
+  'CEIL',
+  'TIMESTAMPADD',
+  'timestamp_expr',
```

Comment:
nope. When you create a new sql parser it accepts an array of strings as functions. In the web-console I do that by pulling all of them from the docs. This is just a list of function I am using as tests and I put that in there as a test at one point.  I will take it out since I don't need it any more though.